acceptance/localcluster/localcluster.go

```diff
@@ -564,9 +564,11 @@ func (l *Cluster) stop() {
 				log.Infof("node %d: stderr at %s", i, file)
 			}
 		} else if crashed {
+			// Don't remove crashed containers - could be useful for post-mortem.
```

Comment:
if you insist, I'd rather close the PR. Adding a flag to that seems not
worth it.

On Fri, Oct 30, 2015 at 2:57 PM, Peter Mattis <notifications@github.com>
wrote:

> In acceptance/localcluster/localcluster.go
> <https://github.com/cockroachdb/cockroach/pull/2974#discussion_r43539887>:
>
> > @@ -564,9 +564,11 @@ func (l *Cluster) stop() {
> >  				log.Infof("node %d: stderr at %s", i, file)
> >  			}
> >  		} else if crashed {
> > +			// Don't remove crashed containers - could be useful for post-mortem.
>
> I think this behavior should be under a flag and default to false or be
> controlled by an env variable.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2974/files#r43539887>.
>